.github/workflows/node.js.yml

```diff
@@ -70,12 +70,8 @@ jobs:
       run: npx playwright-webkit install-deps
       if: matrix.NPM_SCRIPT == 'test:browser' && contains(matrix.BROWSERS, 'WebkitHeadless')
 
-    - name: Build jQuery for Lint
-      run: npm run build:all
-      if: matrix.NPM_SCRIPT == 'test:browserless'
-
     - name: Lint code
-      run: npm run lint
+      run: npm run build:all && npm run lint
```

Comment:
The removed build above was only the first build on Node 20, but I think any of them could have failed. But, I realized lint only runs on Node 18.x, so I moved it there.